change-notes/1.23/analysis-python.md

```diff
@@ -11,4 +11,4 @@
 |-----------|----------|-------------|
 | Clear-text logging of sensitive information (`py/clear-text-logging-sensitive-data`) | security, external/cwe/cwe-312 | Finds instances where sensitive information is logged without encryption or hashing. Results are shown on LGTM by default. |
 | Clear-text storage of sensitive information (`py/clear-text-storage-sensitive-data`) | security, external/cwe/cwe-312 | Finds instances where sensitive information is stored without encryption or hashing. Results are shown on LGTM by default. |
-
+| Binding a socket to all network interfaces (`py/bind-socket-all-network-interfaces`) | security | Finds instances where a socket is bound to all network interfaces. Results are not shown on LGTM by default. |
```

Comment:
This comment says that results aren't shown on LGTM by default, but I thought that high precision errors would be shown. Perhaps I'm wrong about this, or maybe the precision has changed?

Reply:
You are right, changed to show by default.